build.py

```diff
@@ -756,32 +764,42 @@ def dockerfile_prepare_container_linux(argmap, backends):
 # example libcurl only needed for GCS?)
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-         libb64-0d \
-         libcurl4-openssl-dev \
-         libre2-5 && \
+            libb64-0d \
+            libcurl4-openssl-dev \
+            libre2-5 \
+            git \
+            dirmngr \
+            libnuma-dev \
+            curl \
+            {ort_dependencies} && \
     rm -rf /var/lib/apt/lists/*
+'''.format(ort_dependencies=ort_dependencies)
+
+    if enable_gpu:
+        df += install_dcgm_libraries(argmap['DCGM_VERSION'])
+        df += '''
+# Extra defensive wiring for CUDA Compat lib
+RUN ln -sf ${_CUDA_COMPAT_PATH}/lib.real ${_CUDA_COMPAT_PATH}/lib \
+ && echo ${_CUDA_COMPAT_PATH}/lib > /etc/ld.so.conf.d/00-cuda-compat.conf \
+ && ldconfig \
+ && rm -f ${_CUDA_COMPAT_PATH}/lib 
 '''
-    df += install_dcgm_libraries(argmap['DCGM_VERSION'])
+
     # Add dependencies needed for python backend
     if 'python' in backends:
         df += '''
 # python3, python3-pip and some pip installs required for the python backend
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-         python3 libarchive-dev \
-         python3-pip && \
+            python3 libarchive-dev \
```

Comment:
Here too

Reply:
I have commented above, This is to keep all the apt-get indents the same